src/examples/tensorflow/ssd300_demo/README.md

```diff
@@ -16,31 +16,23 @@ For this demo, launch one inf1.xlarge EC2 instance. We recommend using the lates
 
 Please configure your ubuntu16/ubuntu18/yum repo following the steps in the [Neuron installation guide](../../../../docs/neuron-install-guide.md) in order to install `tensorflow-model-server-neuron`.
```

Comment:
this is confusing when we have everything in requirements.txt Can we remove this line ?

Reply:
This is for apt packages.

Reply:
but we have sudo apt install commands in this tutorial as well. User literally only needs to run sudo apt install tensorflow-model-server-neuron

Reply:
There is only one `sudo apt install tensorflow-model-server-neuron` in this tutorial. It is to make this tutorial self-contained. The configuration step in the installation guide only installs runtime packages.

Reply:
Then, we should remove the line 'to install `tensorflow-model-server-neuron`.